test/integration-test.py

```diff
@@ -41,6 +41,7 @@ def run_client_tests():
     run(cmd, cwd=root)
 
 def run_expired_authz_purger():
+    return
```

Comment:
I think you left this disabled by accident

```suggestion
```

Reply:
Yep, fixed.